### PR TITLE
Updated the layout of the dashboard. Added a top menu and modified the sidebar.

### LINKED ISSUE #328: Add a user profile section / menu item to the dashboard.
**Issue/Feature Description:**
In the current dashboard when the user is logged in, there is no user profile section. The logout button is located under a dropdown that has some other features and functions that have no relation to one another.
**Why this issue to fixed / feature is needed(give scenarios or use cases):**
In most dashboards after signing in the user invariably has a profile section where they can see the basic details, edit and save, change password etc. As the current dashboard does not really have all these features currently we can have a section where the user can see the user name, and the current user details. The logout button can be places with this section.

**How to reproduce, in case of a bug:**

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**
/kind enhancement

**What this PR does / why we need it**:
The design of the dashboard with only a sidebar menu limits us to provide links only in the sidebar. There is no top menu and that space can be used more efficiently for links like User Profile, Notifications and any future functionality. 
**Which issue(s) this PR fixes**:

Fixes #328 : There is no User profile as of now but this is the first step in creating a more traditional design.

**Test Report Added?**:
Screenshots added.
**Test Report**:
layout-changed-1
layout-changed-2

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by kumarashit** [comment]
@anvithks please rebase the PR 
proxy changes already merged..
For the top bar color, it seems little odd...Not in contrast with the whole page..Maybe the bottom wave color be put..Just check some better option

**Comment by anvithks** [comment]
The proxy.conf.js changes do not matter as they are required only when setting up local development.
Those values are always changed.

The color of the top menu is trying to make it inline with the overall SODA color scheme (website, logo etc).

I tried different colors (including the wave color) in top menu.
This looked best out of the lot IMO.

**Comment by anvithks** [comment]
@kumarashit Here's a screenshot with the color set to the wave color.
The wave has a transparency applied which I have removed from the menubar.

layout-changed-3

**Comment by kumarashit** [comment]
@anvithks thanks..Looks better

File: proxy.conf.json [hunk 1]
```diff
@@ -1,36 +1,36 @@
 {
```
Comment (kumarashit) [inline proxy.conf.json:1]: Is this file required only for dev env? I see with every update there is IP change and also it's static IP..Can we just ignore?

Comment (anvithks) [reply proxy.conf.json:1]: Yes please ignore.
This file need not be updated.

**Comment by anvithks** [comment]
> @anvithks thanks..Looks better
> 

I will push this in another commit.
It's just a screenshot for now.

**Comment by anvithks** [comment]
@kumarashit I have changed the color of the menubar. Not the same color as the one shared yesterday but a little brighter shade of the same to improve legibility of white over the background. Here is a screenshot for your reference.
layout-changed-4

File: proxy.conf.json (hunk 1 above)
Comment (kumarashit) [reply proxy.conf.json:1]: Ok

Comment (kumarashit) [reply proxy.conf.json:1]: Please remove it from PR..

**Comment by anvithks** [comment]
@kumarashit Removed. Please check. @skdwriting Please review and approve.
